src/bot.ts

```diff
@@ -13,4 +13,13 @@ export const botPullRequestCommentSubcommands: {
 
 export const setupBot = (ctx: Context, bot: Probot): void => {
   setupEvent(ctx, bot, "issue_comment.created", onIssueCommentCreated);
+
+  // test to see whether this even is caught
```

Comment:
Leftover to be removed?

Reply:
no, i'm curious whether this we will see these errors in logs, and I won't be useful, i'll cleanup